src/main/java/com/minecolonies/coremod/util/IItemHandlerToIInventoryWrapper.java

```diff
@@ -0,0 +1,172 @@
+package com.minecolonies.coremod.util;
+
+import com.minecolonies.coremod.inventory.api.IWorldNameableModifyable;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.inventory.IInventory;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.text.ITextComponent;
+import net.minecraftforge.items.IItemHandlerModifiable;
+
+import javax.annotation.Nullable;
+
+/**
+ * Class wrapper for an IItemHandler to an IInventory.
+ */
+public class IItemHandlerToIInventoryWrapper implements IInventory
+{
+    private final IItemHandlerModifiable wrapped;
+    private final IWorldNameableModifyable named;
+
+    public IItemHandlerToIInventoryWrapper(final IItemHandlerModifiable wrapped, final IWorldNameableModifyable named) {this.wrapped = wrapped;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)